app/services/item_change_set_persister.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+# Writes updates Cocina Models
```

Comment:
Also, is this intended only for updating Cocina::Models::DRO (i.e., not APOs or Collections)? If so, can you make it explicit?

Reply:
This class is intended to encapsulate the implementation of the API, so that "DRO" is not relevant from the outside.  All you need to know is that this takes an item and writes it via the API.